starboard/tools/testing/test_runner.py

```diff
@@ -163,12 +164,24 @@ class TestLauncher(object):
 
   def __init__(self, launcher):
     self.launcher = launcher
-    self.runner_thread = threading.Thread(target=self._Run)
 
     self.return_code_lock = threading.Lock()
     self.return_code = 1
 
   def Start(self):
+    if hasattr(self.launcher, "Run2"):
```

Comment:
Conside moving these checks into `AbstractLauncher` member such as

```
class AbstractLauncher(object)
   ...
  def hasExtendedInterface() -> bool:
    return hasattr(self,"Run2") and hasatttr(self,"InitDevice")
  ```
  
  Then the calling code can simply check `if self.launcher.hasExtendedInterface()`

Reply:
Done. Added method `def HasExtendedInterface() -> bool:`.